src/timeline/InputBar.cpp

```diff
@@ -518,6 +560,8 @@ InputBar::showPreview(const QMimeData &source, QString path, const QStringList &
           [this](const QByteArray data, const QString &mime, const QString &fn) {
                   setUploading(true);
 
+                  setText("");
```

Comment:
What if you want to send text with your upload!

Reply:
We don't support that yet. When we do that, it will need more fundamental changes :3